app/controllers/wiki_controller.rb

```diff
@@ -301,15 +301,16 @@ def index
     sort_param = params[:sort]
     order_string = 'node_revisions.timestamp DESC'
 
-    if sort_param == 'title'
-      order_string = 'node_revisions.title ASC'
-    elsif sort_param == 'last_edited'
+    case sort_param
+    when 'title'
+      order_string = 'node.slug ASC'
+    when 'last_edited'
       order_string = 'node_revisions.timestamp DESC'
-    elsif sort_param == 'edits'
+    when 'edits'
       order_string = 'drupal_node_revisions_count DESC'
-    elsif sort_param == 'page_views'
-      order_string = 'views DESC'
-    elsif sort_param == 'likes'
+    when 'page_views'
+      order_string = 'node.vid DESC'
```

Comment:
I think here's the issue.

Reply:
Thanks @gauravano , the GIF is broken so I can't see the behavior but let me give this a second look.